extensions/wikia/AchievementsII/services/AchAwardingService.class.php

```diff
@@ -612,38 +613,38 @@ private function awardNotInTrackBadge( $badge_type_id, $badge_lap = null ) {
 
 		$notInTrackStatic = AchConfig::getInstance()->getNotInTrackStatic();
 
-		if ( isset( $notInTrackStatic[$badge_type_id] ) ) {
-			$badge_level = $notInTrackStatic[$badge_type_id]['level'];
+		if( isset( $notInTrackStatic[ $badge_type_id ] ) ) {
+			$badge_level = $notInTrackStatic[ $badge_type_id ][ 'level' ];
 		} else {
 			$badge_level = BADGE_LEVEL_PLATINUM;
 		}
 
 		$this->mNewBadges[] = array( 'badge_type_id' => $badge_type_id,
-									'badge_lap' => $badge_lap,
-									'badge_level' => $badge_level );
+			'badge_lap' => $badge_lap,
+			'badge_level' => $badge_level );
 
-		if ( !isset( $this->mBadges[$badge_type_id] ) ) {
-			$this->mBadges[$badge_type_id] = 0;
+		if( !isset( $this->mBadges[ $badge_type_id ] ) ) {
+			$this->mBadges[ $badge_type_id ] = 0;
 		}
-		$this->mBadges[$badge_type_id]++;
+		$this->mBadges[ $badge_type_id ]++;
 
-		if ( $badge_type_id == BADGE_WELCOME ) {
-			if ( !isMsgEmpty( 'welcome-user-page' ) ) {
+		if( $badge_type_id == BADGE_WELCOME ) {
+			if( !isMsgEmpty( 'welcome-user-page' ) ) {
 				$userPageTitle = $this->mUser->getUserPage();
-				if ( $userPageTitle ) {
+				if( $userPageTitle ) {
 					$userPageArticle = new Article( $userPageTitle, 0 );
-					if ( !$userPageArticle->exists() ) {
+					if( !$userPageArticle->exists() ) {
 						$userWikia = User::newFromName( 'Wikia' );
 
 						//#60032: forcing IP for bot since this code is run in a real user session and not from a maintenance script
 						$origIP = $wgRequest->getIP();
-						$wgRequest->setIP(  '127.0.0.1'  );
+						$wgRequest->setIP( '127.0.0.1' );
 
 						//fixme. following functionality is done by HAWelcome extension...
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Take the required action to fix the issue indicated by this "FIXME" comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1134)